addons/mail/static/src/models/call_main_view.js

```diff
@@ -149,6 +136,28 @@ registerModel({
                 showOverlayTimer: { doReset: this.showOverlayTimer ? true : undefined },
             });
         },
+        _updateLayout() {
+            if (!this.exists()) {
+                return;
+            }
+            if (!this.component.root.el) {
+                return;
+            }
+            if (!this.tileContainerRef.el) {
+                return;
+            }
```

Comment:
Also all of them should be in the `on` methods.
Again, it doesn't update the layout if not in these conditions, so don't call this method.